### PR TITLE
Update fantomas

### PR DESCRIPTION
**Pull request by CaptnCodr**
- Updates Fantomas to 5.0.0
- Updates FsDocs to 16.1.1
- Build with dotnet 6.0.401

### CODE REVIEW

File: src/FsUnit.Xunit/CustomMatchers.fs [hunk 1]
```diff
@@ -57,8 +57,8 @@ let throw(t: Type) =
             try
                 testFunc()
                 false
-            with
-            | ex -> t.IsAssignableFrom(ex.GetType())
+            with ex ->
+                t.IsAssignableFrom(ex.GetType())
```
Comment (sergey-tihon) [inline src/FsUnit.Xunit/CustomMatchers.fs:61]: @CaptnCodr  do you like such formatting? never saw it  formated  this way

Comment (CaptnCodr) [reply src/FsUnit.Xunit/CustomMatchers.fs:61]: I'm wondered about that is actual legal. But I kinda like it. 
That is also mentioned here: https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-trywith-expressions

Comment (sergey-tihon) [reply src/FsUnit.Xunit/CustomMatchers.fs:61]: Then it is good to go

**Comment by CaptnCodr** [comment]
@sergey-tihon Is releasing a new version really needed? Nothing happened to the functionality. 
Just ship it with the next change?

**Comment by sergey-tihon** [comment]
No, would be enough just to merge it

**Pull request merged**